src/mainboard/emulation/qemu-armv7/src/main.rs

```diff
@@ -24,9 +23,10 @@ pub extern "C" fn _start() -> ! {
     uart_driver.pwrite(b"Welcome to oreboot\r\n", 0).unwrap();
     let s = &mut [uart_driver];
     let console = &mut DoD::new(s);
+    let mut w = print::WriteTo::new(console);
+    let spi = SliceReader::new(DTB);
 
-    if let Err(err) = print_fdt(console) {
-        let mut w = print::WriteTo::new(console);
+    if let Err(err) = print_fdt(&spi, &mut w) {
         write!(w, "error: {}\n", err).expect(err);
```

Comment:
I'm surprised that clippy isn't suggesting `writeln!`.

Reply:
Clippy is not enabled on all crates yet. There is pretty long list of BROKEN_CRATES_TO_CLIPPY in the Makefile (I am starting with #319), which is why this was not caught.